splunk-on-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,555 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "metadata": {
+        "description": "Location where resources will be provisioned"
+      }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "metadata": {
+        "description": "Storage account type which determines data redundancy and underlying drive type"
+      }
+    },
+    "deploymentSize": {
+      "type": "string",
+      "defaultValue": "Standalone",
+      "allowedValues": [
+        "Standalone",
+        "Cluster"
+      ],
+      "metadata": {
+        "description": "Splunk deployment type"
+      }
+    },
+    "domainName": {
```

Comment:
We see a bunch of errors from this. So it;s better to call this a prefix to avoid any confusion as to the format. See #966